properties/P000104.md

```diff
@@ -10,12 +10,14 @@ refs:
   name: On semi-metric spaces (Wilson)
 ---
 
+A space whose topology is generated by a *network* of
+symmetric balls $B(x,\varepsilon)=\{y\in X:d(x,y)<\varepsilon\}$ for some symmetric $d$.
+That is, a subset $U \subseteq X$ is open iff for each $x \in U$,
+there is some $\varepsilon > 0$ with $B(x,\varepsilon)\subseteq U$.
+
 For a set $X$, a function $d:X^2\to[0,\infty)$ is called a *symmetric*
 if it satisfies reflexivity ($d(x,y)=0\Leftrightarrow x=y$) and symmetry
-($d(x,y)=d(y,x)$). A space $X$ is called *symmetrizable* if
-there is a symmetric such that a subset $U \subseteq X$ is open iff for each $x \in U$,
-there is some $\varepsilon > 0$ with $B(x,\varepsilon)=\{y\in X:d(x,y)<\varepsilon\}$
-being contained in $U$.
+($d(x,y)=d(y,x)$); in this case, we will call this symmetric a *semimetric*.
```

Comment:
I think you have mistakenly put this line in symmetrizable instead of semimetrizable?